pulsar-client-admin-api/pom.xml

```diff
@@ -0,0 +1,73 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.pulsar</groupId>
+        <artifactId>pulsar</artifactId>
+        <version>2.8.0-SNAPSHOT</version>
+        <relativePath>..</relativePath>
+    </parent>
+
+    <artifactId>pulsar-client-admin-api</artifactId>
+    <name>Pulsar Client Admin :: API</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>pulsar-common</artifactId>
+            <version>${project.parent.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>pulsar-client-original</artifactId>
+            <version>${project.parent.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>pulsar-package-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Is it intentional that these dependencies are included? This change seems to be causing a major regression to Pulsar IO connectors. Please see #9572 . /cc @sijie @eolivelli

Reply:
The size of Pulsar IO connectors files get back to normal after changing the scope of `pulsar-client-admin-api` in `pulsar-functions/api-java/pom.xml` from `<scope>compile</scope>` to `<scope>provided</scope>`.

However I'm a bit concerned about the usefulness of the new pulsar-client-admin-api library since it's not really a standalone api dependency. It pulls in all dependencies. WDYT?

Reply:
Size of https://hub.docker.com/layers/streamnative/pulsar-all/2.8.0-rc-202102022225/images/sha256-50d6a03312262d5b60efbb29e8c4a3f2245d1317ee25164e18bbd43a0d86113c?context=exploreis 2.91GB , 
on the previous day, it was 2.25GB:
https://hub.docker.com/layers/streamnative/pulsar-all/2.8.0-rc-202102012227/images/sha256-04d33735218a612dd53c7963a35bbe9d8c9a0db1625555aa501cfa95821e029f?context=explore

![image](https://user-images.githubusercontent.com/66864/107939259-282d5c80-6f8f-11eb-8f4f-aa430543bf34.png)

The ≈700MB increase most likely happened after this PR was merged.